config/config.go

```diff
@@ -36,8 +36,9 @@ type Config struct {
 	}
 
 	Auth struct {
-		RefererURLs  []string `info:"Allowed referer URLs for auth and redirects."`
-		DisableBasic bool     `public:"true" info:"Disallow username/password login."`
+		RefererURLs     []string `info:"Allowed referer URLs for auth and redirects."`
+		DisableBasic    bool     `public:"true" info:"Disallow username/password login."`
+		DisableAuthLink bool     `public:"true" info:"Disallow linking mobile devices."`
```

Comment:
nit:
```suggestion
		DisableAuthLink bool     `public:"true" info:"Disallow linking external devices."`
```

because I _know_ we can get this working in a Tesla 😉

Reply:
Haha I'll make it a bit more generic, but I want it to be clear that it is used/required for the mobile app